airflow/migrations/versions/2e541a1dcfed_task_duration.py

```diff
@@ -24,7 +24,7 @@
 Create Date: 2015-10-28 20:38:41.266143
 
 """
-
+# pylint: disable=missing-docstring,no-member,too-many-nested-blocks,wrong-import-order,ungrouped-imports
```

Comment:
Why? In my opinion, only the first two options should be turned off. Others should be able to be fixed in this file.

Reply:
> they are automatically generated by alembic

That's also the point why I didn't comment on those.
I think it is similar to the `_vendor` code - it is not manually written by one of us.